packages/datatrak-web/src/views/AccountSettingsPage/ChangePasswordSection/ChangePasswordForm.tsx

```diff
@@ -87,6 +91,8 @@ export const ChangePasswordForm = () => {
     onSuccess: () => successToast('Your password has been successfully updated'),
   });
 
+  const submissionShouldBeDisabled: boolean = isValidating || !isValid || isSubmitting;
```

Comment:
Happy for this to stay, but in cases like this where the type is implied from the values, you probably don't need to explicitly state the type